.circleci/config.yml

```diff
@@ -5,12 +5,16 @@ commands:
       - run:
           name: Js install + Build
           command: |
-            # install npm + yarn + wasm-pack
-            sudo apt install npm
+            # install
+            # node
+            curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+            sudo apt-get install -y nodejs
+            # yarn
             curl -o- -L https://yarnpkg.com/install.sh | bash
             export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
+            # wasm-pack
             curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
-            # build
+            # buidl
```

Comment:
build?

Reply:
this one is a feature: https://academy.binance.com/glossary/buidl